diff --git a/../rumqttd-old/config/rumqttd.conf b/demo.toml
index ce704a6..2084199 100644
--- a/../rumqttd-old/config/rumqttd.conf
+++ b/demo.toml
@@ -1,4 +1,3 @@
-# Broker id. Used to identify local node of the replication mesh
 id = 0
 
 # A commitlog read will pull full segment. Make sure that a segment isn't
@@ -6,40 +5,57 @@ id = 0
 # latencies of other connection. Not a problem with preempting runtimes
 [router]
 id = 0
-dir = "/tmp/rumqttd"
+instant_ack = true
 max_segment_size = 10240
 max_segment_count = 10
+max_read_len = 10240
 max_connections = 10001
 
 # Configuration of server and connections that it accepts
-[servers.1]
+[v4.1]
+name = "v4-1"
 listen = "0.0.0.0:1883"
 next_connection_delay_ms = 1
-    [servers.1.connections]
-    connection_timeout_ms = 5000
+    [v4.1.connections]
+    connection_timeout_ms = 60000
     max_client_id_len = 256
     throttle_delay_ms = 0
-    max_payload_size = 5120
-    max_inflight_count = 200
+    max_payload_size = 20480
+    max_inflight_count = 500
     max_inflight_size = 1024
+    dynamic_filters = true
 
-# Configuration of server and connections that it accepts
-[servers.2]
+Example configuration for a TLS enabled server
+[v4.2]
+name = "v4-2"
 listen = "0.0.0.0:8883"
 next_connection_delay_ms = 10
-    # Cert config
-    [servers.2.cert]
-    cert_path = "tlsfiles/server.cert.pem"
-    key_path = "tlsfiles/server.key.pem"
-    ca_path = "tlsfiles/ca.cert.pem"
-    # Connection parameters
-    [servers.2.connections]
-    connection_timeout_ms = 5000
+    # tls config for rustls
+    [v4.2.tls]
+    certpath = "./localhost.cert.pem"
+    keypath = "./localhost.key.pem"
+    capath = "./ca.cert.pem"
+    # settings for all the connections on this server
+    [v4.2.connections]
+    connection_timeout_ms = 60000
+    throttle_delay_ms = 0
+    max_payload_size = 20480
+    max_inflight_count = 100
+    max_inflight_size = 1024
+
+[v5.1]
+name = "v5-1"
+listen = "0.0.0.0:1884"
+next_connection_delay_ms = 1
+    [v5.1.connections]
+    connection_timeout_ms = 60000
     max_client_id_len = 256
     throttle_delay_ms = 0
     max_payload_size = 5120
     max_inflight_count = 100
     max_inflight_size = 1024
 
+[ws]
+
 [console]
 listen = "0.0.0.0:3030"
